content/posts/2022-features.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "2022 for Packit"
+date: 2023-01-28T11:58:50+01:00
+---
+
+## Packit project in 2022
+
+As you will see in the following paragraphs, the year 2022 was really fruitful for the Packit project. Without further ado, let’s take a look at what the Packit team accomplished last year:
+
+## Fedora automation
+
+Packit team has made a huge improvement in downstream automation. At the beginning of the year, we [finished the workflow](https://packit.dev/posts/downstream-automation/) and you are now able to use Packit to get your release from upstream via dist-git and Koji to Bodhi. As usual, you can pick just what you need. This workflow consists of three jobs:
```

Comment:
```suggestion
We have made a huge improvement in downstream automation. At the beginning of the year, we [finished the workflow](https://packit.dev/posts/downstream-automation/) and you are now able to use Packit to get your release from upstream via dist-git and Koji to Bodhi. As usual, you can pick just what you need. This workflow consists of three jobs:
```

"Packit team" term is used in 2 sentences in a row, so I'm shortening the 2nd use.